mindone/models/dit.py

```diff
@@ -6,17 +6,29 @@
 import numpy as np
 
 import mindspore as ms
+import mindspore.numpy as msnp
 from mindspore import Parameter, Tensor, nn, ops
 from mindspore.common.initializer import Constant, Normal, One, XavierNormal, XavierUniform, Zero, initializer
 
-from ..utils.version_control import check_valid_flash_attention, choose_flash_attention_dtype
+from ..utils.version_control import MSVersion, check_valid_flash_attention, choose_flash_attention_dtype
 
 logger = logging.getLogger(__name__)
-FLASH_IS_AVAILABLE = check_valid_flash_attention()
-if FLASH_IS_AVAILABLE:
-    from mindspore.nn.layer.flash_attention import FlashAttention
 
+# try import fa
+try:
```

Comment:
somehow it looks dirty due to frequently change of API in Mindspore. How about make this import things into a separate wrapper? it will look clear

Reply:
Agreed, it's a good idea to introduce MindONE-wise `FlashAttention` that is MindSpore version-agnostic.